src/org/mozilla/javascript/ScriptRuntime.java

```diff
@@ -733,6 +743,148 @@ public static double toNumber(String s) {
         }
     }
 
+    /**
+     * Convert the value to a BigInt.
+     */
+    public static BigInteger toBigInt(Object val)
+    {
+        for (;;) {
+            if (val instanceof BigInteger)
```

Comment:
I'd like to get to a point where we start to apply a formatting standard to the code. I think that one thing that we'll want to enforce is using brackets with conditional statements, even one-line ones. So this is a nit but please consider it.

Reply:
I too like the writing style you suggested.
I want to automatically check what kind of writing style I should use. ref. #863

Reply:
Apparently, the current coding style allows one-line. I have checked and corrected what I have added.
https://github.com/mozilla/rhino/blob/a25c5176d37fad4da77e97a2e89953258801293e/src/org/mozilla/javascript/ScriptRuntime.java#L413-L415